README.md

```diff
@@ -7,9 +7,28 @@ GIFramework Maps
 
 GIFramework Maps is a [.NET](https://dot.net) based web mapping application designed and developed by the Dorset Council GIS team.
 
-It is used by Dorset Council as their primary web mapping application for staff and the public. [You can see it running live here](https://gi.dorsetcouncil.gov.uk/dorsetexplorer).
+GIFramework Maps is used by Dorset Council as their primary web mapping application for staff and the public. [You can see it running live here](https://gi.dorsetcouncil.gov.uk/dorsetexplorer).
+
+GIFramework Maps is Open Source software (using [OpenLayers](https://openlayers.org/) and [Bootstrap](https://getbootstrap.com/), as well as many other libraries) and is licensed under the MIT licence.
+
+## Features
+
+GIFramework Maps is a web application for creating, sharing and managing web maps using data from multiple OGC web services. It is designed to be user friendly yet powerful, with a raft of included features.
+
+With GIFramework Maps, you can:
+- Display layers from multiple sources, including raster and vector data
+- Query spatial data with ease, and customise what users see with nunjucks templating
+- Add, edit and export annotations
+- Measure lines and areas
+- Export maps to PDFs with customisable templates and legends
+- Share maps using a simple link
+- Track your location using your GPS, and export the track
+- Search for locations using mutliple configurable search services
+- Manage all your maps using the built-in GUI administation pages
```

Comment:
Typo - administration

Reply:
My approval is subject to fixing this one that Lucy has spotted as I'm sure you have guessed :-)